docs/E-Secondary-Storage/text-files.md

```diff
@@ -1,4 +1,6 @@
 ---
+id: text-files
+title: Text Files
 sidebar_position: 1
 ---
```

Comment:
Add a description here, check the standard https://github.com/Seneca-ICTOER/IPC144/pull/143